recipe/meta.yaml

```diff
@@ -36,8 +36,11 @@ requirements:
 test:
   source_files:
     - tests
+    - pyproject.toml
```

Comment:
Why do you need `pyproject.toml` in tests?

Reply:
pytest reads test configuration from there when available.

Reply:
Hmm.  So we should copy it for tests if it exists (and possibly some other conditions)?

Reply:
The relevant part here was this:
https://github.com/jupyter/jupyter_core/blob/v5.7.2/pyproject.toml#L112-L129

Sometimes it's worth taking, sometimes not. Sometimes that configuration is found in other files (setup.cfg, conftest.py...) https://docs.pytest.org/en/7.1.x/reference/customize.html